pages/02-events.md

```diff
@@ -2,9 +2,9 @@
 title: Events
 ---
 
-## SPLS 
+## SPLS
 
-The [Scottish Programming Languages Seminar](https://spls-series.github.io/) (SPLS) is a regular event for discussion of all aspects of programming languages.
+The [Scottish Programming Languages Seminar](https://scottish-pl-institute.github.io/spls/) (SPLS) is a regular event for discussion of all aspects of programming languages.
```

Comment:
We only need a relative link! :fireworks: 
```suggestion
The [Scottish Programming Languages Seminar](spls/) (SPLS) is a regular event for discussion of all aspects of programming languages.
```

Reply:
Oh, of course...